### PR TITLE
Requesito 11 completo version 1.0

### PR DESCRIPTION
**Pull request by Junior030**
- [X] Req. 11 ✅

### CODE REVIEW

**Review by gabrielmirandaBR — CHANGES_REQUESTED** [review]
Bom dia @Junior030, beleza? Segue minha sugestão de review!

File: src/components/pages/ItemProduct.jsx [hunk 1]
```diff
@@ -22,41 +26,116 @@ export default class ItemProduct extends Component {
...
 
+  evaluationData({ target: { name, value } }) {
+    this.setState({
+      [name]: value,
+    });
+  }
+
   render() {
-    const { item } = this.state;
-    const { getProduct } = this.props;
```
[the remark this answers is not in the document]
Comment (Junior030) [reply src/components/pages/ItemProduct.jsx:27]: Resolving conflicts on CardBtn

File: src/components/pages/ItemProduct.jsx [hunk 2]
```diff
@@ -22,41 +26,116 @@ export default class ItemProduct extends Component {
...
-          <button
-            data-testid="product-detail-add-to-cart"
-            type="submit"
-            onClick={ () => getProduct(item) }
-          >
-            Adicionar ao carrinho
-          </button>
-        </div>
-      </>
+      <main>
```
[the remark this answers is not in the document]
Comment (Junior030) [reply src/components/pages/ItemProduct.jsx:39]: Resolving conflicts on CardBtn

File: src/components/pages/ItemProduct.jsx [hunk 3]
```diff
@@ -22,41 +26,116 @@ export default class ItemProduct extends Component {
...
+      </main>
     );
   }
 }
+
 ItemProduct.propTypes = {
   match: PropTypes.shape({
     params: PropTypes.shape({ id: PropTypes.string }),
   }).isRequired,
-  getProduct: PropTypes.func.isRequired,
```
[the remark this answers is not in the document]
Comment (Junior030) [reply src/components/pages/ItemProduct.jsx:61]: Resolving conflicts on CardBtn

File: src/components/pages/ItemProduct.jsx [hunk 4]
```diff
@@ -22,41 +26,116 @@ export default class ItemProduct extends Component {
...
+            <strong data-testid="product-detail-name">{item.title}</strong>
+          </div>
+          <div>
+            Preço:
+            {' '}
+            <strong>{item.price}</strong>
+          </div>
+          <img src={ item.thumbnail } alt="imagem do produto" />
+        </section>
+        <form>
```
[the remark this answers is not in the document]
Comment (Junior030) [reply src/components/pages/ItemProduct.jsx:65]: Resolving conflicts on CardBtn

File: src/components/pages/ItemProduct.jsx [hunk 5]
```diff
@@ -22,41 +26,116 @@ export default class ItemProduct extends Component {
...
+      [name]: value,
+    });
+  }
+
   render() {
-    const { item } = this.state;
-    const { getProduct } = this.props;
+    const { item, email, menssage, rating } = this.state;
     return (
-      <>
```
[the remark this answers is not in the document]
Comment (Junior030) [reply src/components/pages/ItemProduct.jsx:29]: Resolving conflicts on CardBtn

**Review by Junior030 — COMMENTED** [review]
Resolving conflicts on CardBtn

**Review by gabrielmirandaBR — APPROVED** [review]
Valeu demais!!! 🚀

**Review by Lotar-lucas — APPROVED** [review]

**Pull request merged**